src/App.tsx

```diff
@@ -1,96 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { Select } from './components/Select';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const items = getNumbers(1, 42).map(n => `Item ${n}`);
+
+function itemsPerPage(perPageNum: number) {
+  const newItems = [...items];
+  const resultItems: Array<string>[] = [];
+
+  newItems.map((_item, i) => {
```

Comment:
The 'map' function is not used correctly here. It is used to transform every element of the original array into something else, returning a new array. Here, you are using it like 'forEach' or 'reduce'. You should replace it if you don't want to transform the array elements.